crates/uniswapx-rs/src/order.rs

```diff
@@ -172,9 +221,9 @@ impl V2DutchOrder {
 
                 // add exclusivity override to amount
                 if self.cosignerData.decayStartTime.gt(&timestamp) && !self.cosignerData.exclusiveFiller.is_zero() {
-                    let exclusivity = self.cosignerData.exclusivityOverrideBps.wrapping_add(Uint::from(10000));
+                    let exclusivity = self.cosignerData.exclusivityOverrideBps.wrapping_add(BPS);
```

Comment:
actually... is wrapping the proper behavior here? maybe better to do checked arithmetics and throw on error?

Reply:
yeah probably should throw

Reply:
Yeah looks like we do wrapped arithmetic all over but I added checks for some of it.